### DISCUSSION TITLE
Portable software

### DISCUSSION THREAD

**Discussion by PiliJ** [NexovaDev/UpdateHub#450] [Ideas]
Hi,
Thank you for the soft !
For my part, I regret that it cannot take into consideration the location where portable software is located (in my case, on drive D:)

**Comment by onyx-nxt** [comment]
Like upgrading apps? Or where UpdateHub installs?

**Reply by PiliJ** [reply]
"Like upgrading apps?" : Yes
"Or where UpdateHub installs?" : Yes
But j'aimerais mettre mes apps portables à jour aussi, comme le proposait Sumo par exemple.
But I'd like to be informed about updates for my portable apps too.

**Comment by Aardvogel** [comment]
UpdateHub needs to allow you to specify an additional location for Program Files, not everyone installs programs to their Windows drive. So would be good to allow user to specify eg F:\Program Files F:\Program Files (x86) in Settings
